example/quicklook.cpp

```diff
@@ -0,0 +1,113 @@
+//
+// Copyright (c) 2022 Alan Freitas (alandefreitas@gmail.com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+// Official repository: https://github.com/CppAlliance/url
+//
+
+#include <boost/url.hpp>
+#include <iostream>
+
+using namespace boost::urls;
+
+int
+main(int, char**)
+{
+    result<url_view> r = parse_uri( "https://user:pass@www.example.com:443/path/to/my%2dfile.txt?id=42&name=John%20Doe#page%20anchor" );
```

Comment:
This contains all snippets we present in 2.0.quicklook.qkb. You probably want to organize this differently though.